CONTRIBUTING.md

```diff
@@ -148,7 +148,7 @@ cargo run --example 2>&1 \
    -e 'ws_auth' \
    -e 'connect_builtin' \
    -e 'subscribe_logs' \
-   | xargs -I {} sh -c 'if cargo run --example {} --quiet 1>/dev/null; then \
```

Comment:
Unfortunately the `-S` flag does not exist on Linux, I'll add a Bash script so it can more easily be ran across all operating systems

Reply:
Ok, I thought this would be easier, but it didn't count with that; let me know if you need help 💪 
You can go ahead and close this PR if you plan to open another one or an issue in the future

Reply:
Closing for now, will think of a better way to enable contributors to run the examples easily. Bash is quite brittle as well.